src/components/Modal/Modal.tsx

```diff
@@ -59,13 +55,40 @@ export const Modal = ({
     return () => {
       document.removeEventListener("keydown", keyDown);
     };
-  }, [close]);
+  });
+
+  const modalRef = React.useRef();
+  const handleTabKey = (e) => {
+    const focusableModalElements = modalRef.current.querySelectorAll(
+      'a[href], button, textarea, input[type="text"], input[type="radio"], input[type="checkbox"], select'
```

Comment:
IIRC you can also focus any element with a `tabindex`. You can also focus anything with `contentEditable`.
This also doesn't include disabled elements.

I found this SO answer that might provide a more robust selector: https://stackoverflow.com/a/30753870/3732840